recipes/semver200/all/conanfile.py

```diff
@@ -0,0 +1,79 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.files import apply_conandata_patches, copy, get
+from conan.tools.build import check_min_cppstd
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
+import os
+
+
+required_conan_version = ">=1.50.0"
+
+
+class SemVer200Conan(ConanFile):
+    name = "semver200"
```

Comment:
I am curious where the name comes from 🤔 the project class itself `semver` https://github.com/zmarko/semver/blob/a3dff8d19a5a4b10dea5cb3e0998fc8b411648d5/CMakeLists.txt#L2 and I did not find any other reference to it

I'd love to learn the significance of this name 🙏

Reply:
https://github.com/zmarko/semver/tree/a3dff8d19a5a4b10dea5cb3e0998fc8b411648d5#usage

It means semver 2.0.0. I would suggest author-semver as package name, but it seems like semver200 is its unique name.

Reply:
@szmyd Could you please rename the package to zmarok-semver. Need to update the name attribute and folder name

Reply:
@uilianries sure thing :+1:

Reply:
@szmyd thank you!